source/jormungandr/jormungandr/scenarios/ridesharing/ridesharing_service.py

```diff
@@ -67,11 +77,37 @@ def _request_journeys(self, from_coord, to_coord, period_extremity, limit=None):
     @abc.abstractmethod
     def _get_feed_publisher(self):
         """
-
         :return: Rs_FeedPublisher
         """
         pass
 
+    def _get_rs_id(self):
+        return '{}_{}'.format(repr(self.system_id), repr(self.network))
+
+    def record_internal_failure(self, message):
```

Comment:
Not used (yet?) but it's the same format as the one used in ridesharing_helper.py: https://github.com/CanalTP/navitia/pull/2332/files#diff-6bb71b6b8055d0f038b2202dcd00b909R89
So exposed it for future need and consistency.